package.json

```diff
@@ -96,13 +96,15 @@
         "@babel/template": "^7.1.2",
         "@babel/traverse": "^7.1.0",
         "@babel/types": "^7.1.2",
+        "@types/prettier": "^1.15.2",
         "gitignore-globs": "^0.1.1",
         "gitignore-to-glob": "^0.3.0",
         "glob": "^7.1.3",
         "line-column": "^1.0.2",
         "mkdirp": "^0.5.1",
         "prepend": "^1.0.2",
         "prepend-file": "^1.3.1",
+        "prettier": "^1.15.3",
```

Comment:
Not handling the formatting is by design. Formatting is very opinionated and every code base has its own. I think we should lets IDEs handle that

Reply:
How is it handled by the IDE? And I'm pretty sure prettier can pick up the workspaces config. E.g from tslint, eslint ect

Reply:
How about
```glean.prettier: true``` with ```default:false```

Reply:
the problem is that this still doesnt solve the issue of venturing into the formatting territory. I dont see much point in that. Lets folks run their local formatter of their choice. We dont need to do it for the IDE